packages/integrations/stores/ini-file/src/ini-file.ts

```diff
@@ -0,0 +1,44 @@
+import _ from 'lodash';
```

Comment:
`lodash` is missing from this integration's `package.json`. Could you please add it? N.B. it's pinned elsewhere to `^4.17.21', so I'd keep to that version.